src/riak_kv_pb_object.erl

```diff
@@ -190,8 +193,8 @@ process(#rpbputreq{bucket=B0, type=T, key=K, vclock=PbVC,
     case Result of
         consistent ->
             process(Req#rpbputreq{if_not_modified=undefined,
-                                  if_none_match=consistent},
-                    State);
+                                  if_none_match=undefined},
```

Comment:
Yeah good call, I'll add some comments. I didn't think it was that weird at first because there are other case clauses in the same function where we set `if_none_match` and `if_not_modified` to `undefined` before recursing, but I can see how it's more confusing in this particular case.